tests/functional/spec/dates.spec.js

```diff
@@ -15,18 +15,23 @@ describe('Date checks', function() {
 
```

Comment:
I may have missed it, but I can't see a test for when the 'Please select' placeholder isn't in the month dropdown for mandatory dates once a month is selected.